### ISSUE TITLE
Can't use ctrl + r (history search) in a zsh shell spawned by ranger.

### ISSUE THREAD

**Issue by GijsTimmers** [ranger/ranger#1121]
#### Runtime Environment

- Operating system and version: ubuntu 17.10
- Terminal emulator and version: rxvt-unicode (urxvt) v9.22 - released: 2016-01-23
- Python version: Python version: 2.7.14 (default, Sep 23 2017, 22:06:14) [GCC 7.2.0]
- Ranger version/commit: ranger-master 1.9.1
- Locale: None.None

#### Current Behavior
Pressing ctrl + r does nothing in a zsh shell spawned by `ranger` with `:shell zsh`

#### Expected Behavior
have `bck-i-search:` pop up, be able to find previously entered commands

**Comment by toonn** [comment]
I can't reproduce this, both bash and fish can access their history when launched this way. Did you actually start typing to start the reverse search?
Btw, there's the "S" mapping for this.

**Comment by GijsTimmers** [comment]
yes... I can confirm that bash works normally this way. might be a zsh issue.

**Comment by toonn** [comment]
Could you try `:shell zsh -f`? Do you use oh-my-zsh or similar?

**Comment by GijsTimmers** [comment]
Tried that. Same behaviour. I'm not using oh-my-zsh or any similar software.

**Comment by toonn** [comment]
You can list your key bindings with `bindkey -L`, when I do `:shell zsh -f` `^R` is apparently bound to `redisplay`, not sure what that does since it's not a command and there's no manpage.
So I guess you'll have to bind the mapping in your `.zshrc`: `bindkey '^R' history-incremental-search-backward`
Not sure why it would work in a regular shell in urxvt but not a shell in ranger in a shell in urxvt.
